src/deepqmc/train.py

```diff
@@ -91,7 +93,7 @@ def train(  # noqa: C901
                 passed in :data:`opt_kwargs`.
             - :data:`None`: no optimizer is used, e.g. the evaluation of the Ansatz
                 is performed.
-        sampler (~deepqmc.sampling.Sampler): a sampler instance
+        samplers (~deepqmc.sampling.Sampler): a sampler instance
```

Comment:
changed in docs but not in the code?

Reply:
I was going back and forth, whether we need to rename this to `samplers`. In the end, I think it's better to keep it as `sampler`. I've corrected this docstring in https://github.com/deepqmc/deepqmc/commit/b90851c621f8d6c8ef1dfac555e66607f7e153d2.